index.d.ts

```diff
@@ -138,7 +138,15 @@ declare module 'react-native-callkeep' {
      * @description setMutedCall method is available only on iOS.
      */
     static setMutedCall(uuid: string, muted: boolean): void
+  
+    /**
+     * @description toggleAudioRouteSpeaker method is available only on Android.
+     * @param uuid 
+     * @param routeSpeaker 
+     */
+    static toggleAudioRouteSpeaker(uuid: string, routeSpeaker: boolean) {
 
+    }
```

Comment:
Consistency with other types.

```suggestion
    static toggleAudioRouteSpeaker(uuid: string, routeSpeaker: boolean): void
```